services/app-api/forms/naaar.json

```diff
@@ -613,17 +613,17 @@
               "hint": "Enter the provider type the standard applies to. If you wish to further specify the provider type, select the core type most suitable and then add the more specific provider type under “Additional specialty details”, which will appear as an optional text field after the provider type is selected. Do not use this section to indicate populations associated with this provider type (e.g., adult, pediatric, LTSS); these details should be entered in the dedicated population fields below.",
               "choices": [
                 {
-                  "id": "standard_UZK4hxPVnuYGcIgNzYFHCk",
+                  "id": "UZK4hxPVnuYGcIgNzYFHCk",
```

Comment:
do we need to update the seeds for these changes?

Reply:
not for all of them, I just updated it with the required changes! thank you 🙏🏽